src/Defs.h

```diff
@@ -384,11 +384,18 @@ extern TicksTime g_cycleStartTime;
 
 extern debug_level_t g_debug_level;
 
-#ifdef  USING_VMA_EXTRA_API
-extern unsigned char* g_pkt_buf;
-extern struct vma_packets_t* g_pkts;
-extern unsigned int g_pkt_index;
-extern unsigned int g_pkt_offset;
+#ifdef USING_VMA_EXTRA_API
+typedef struct zero_copy_thread_data {
```

Comment:
_thread_ should not be part of the new type.  the TLS thing should be determined upon usage and is not appropriate in the type name.
(Also consider removing the the term _zero_copy_ from the type name.  this type is essentially a data-buffer regardless of its current specific usage.  If you want to relate it to vma, then you can call it vma_data_buffer)